functions/Get-DbaAgentJobOutputFile.ps1

```diff
@@ -0,0 +1,144 @@
+﻿Function Get-DbaAgentJobOutPutFile
```

Comment:
This is a Get- command, please make it test all OutPut files by default unless you see a reason not to. Even if a machine has 300 jobs with 3 steps each, 1000 rows is nbd for PS